modules/flowable-engine/src/test/java/org/flowable/engine/test/bpmn/usertask/UserTaskTest.java

```diff
@@ -281,6 +281,20 @@ public void testCreateUserTaskInterceptor() throws Exception {
         }
     }
 
+    @Test
```

Comment:
FWIW, this file is corrected to assertj format in this [PR](https://github.com/flowable/flowable-engine/pull/2463l).  It is the last file in the listing but the PR hasn't been committed yet.

Reply:
Updated it to use fluent assertions (same as in HumantaskTest).